### PR TITLE
Add rule for SLES-12-030365

### PR DESCRIPTION
**Pull request by anivan-suse**
#### Description:

- SLES-12-030365, cce@sle12: CCE-83248-5

#### Rationale:

- sysctl_net_ipv6_conf_default_forwarding

### CODE REVIEW

**Comment by cipherboy** [comment]
@anivan-suse If you rebase this, I can go ahead and merge it. :-)

**Comment by jan-cerny** [comment]
@anivan-suse Please remove the unrelated commits from the PR. You can do that by rebasing your local branch on the top of the current (updated) upstream master and pushing force to the feature branch in your GitHub fork.

**Comment by jan-cerny** [comment]
@anivan-suse I can see that you have tried to rebased it somehow, but I still can see various "Merge ..." commits in the list of the commits of this PR. Please remove them, a PR with "Merge" or "Fix conflicts" or similar commits can't be accepted.

**Comment by anivan-suse** [comment]
Hey @jan-cerny, thanks for the info! I tried to rebase following [this](https://github.com/ComplianceAsCode/content/pull/7164#issuecomment-870686684) comment I had previously, but it did not get rid of the Merge commits. Do you know how I might best be able to get rid of them? Thanks!

**Comment by jan-cerny** [comment]
> Hey @jan-cerny, thanks for the info! I tried to rebase following [this](https://github.com/ComplianceAsCode/content/pull/7164#issuecomment-870686684) comment I had previously, but it did not get rid of the Merge commits. Do you know how I might best be able to get rid of them? Thanks!

Hi @anivan-suse , you can do similar steps as in the comment you linked, but use an interactive rebase using the `-i` option instead of the automated rebase . When you run `git rebase -i upstream/master`, your editor will open and you will be able to remove the commits.  I think that this article explains it well: https://hackernoon.com/beginners-guide-to-interactive-rebasing-346a3f9c3a6d

File: linux_os/guide/system/network/network-ipv6/configuring_ipv6/sysctl_net_ipv6_conf_default_forwarding/rule.yml [hunk 1]
```diff
@@ -15,13 +15,15 @@ severity: medium
...
     cce@sle15: CCE-85725-0 
+    cce@sle12: CCE-83248-5
 
 references:
     disa: CCI-000366
     nist: CM-6(b),CM-6.1(iv)
     srg: SRG-OS-000480-GPOS-00227
     stigid@sle12: SLES-12-030363
     stigid@sle15: SLES-15-040382
+    stigid@sle12: SLES-12-030365
```
Comment (Mab879) [inline linux_os/guide/system/network/network-ipv6/configuring_ipv6/sysctl_net_ipv6_conf_default_forwarding/rule.yml:26]: This needs to be sorted and please remove the extra `stigid@sle12`.

**Review by Mab879 — CHANGES_REQUESTED** [review]

**Review by Mab879 — APPROVED** [review]
Thank you!

**Pull request merged**